### DISCUSSION TITLE
Show endpoint in model dropdown

### DISCUSSION THREAD

**Discussion by cduk** [open-webui/open-webui#7016] [Ideas]
In the model drop down, it is not possible to see the endpoint that hosts the model:

![image](https://github.com/user-attachments/assets/7b3b9735-d4d4-4346-bc29-3f56093dce6e)

Where the same model is hosted by many different hosts with different characteristics (e.g. speed with Groq or cost) it would be useful to differentiate. This can be done by:

* Having a custom icon applicable for each endpoint; and
* Having tooltip on hover to the external link giving the endpoint URL

**Comment by avviano** [comment]
Actually, this is already possible and very easy to setup. Simply go into your "Admin ► Settings ► Connections ► API Settings" - look for the "PREFIX" field under the API, then add a text designator in the PREFIX field (eg OpenAI or Anthropic or Nvidia or OpenRouter aso). These prefixes will then show up before the model name in your chat model dropdown. 

I do wish the endpoint would show up in the individual model settings / preferences. Right now its pretty hidden.
